src/main/java/com/amazon/opendistroforelasticsearch/sql/plugin/RestSqlAction.java

```diff
@@ -209,7 +210,7 @@ private boolean isSQLFeatureEnabled() {
         return allowExplicitIndex && isSqlEnabled;
     }
 
-    private static ColumnTypeProvider performAnalysis(String sql) {
+    public static ColumnTypeProvider performAnalysis(String sql) {
```

Comment:
why change to public?

Reply:
outdated change from when it was called twice- will revert